index.html

```diff
@@ -258,6 +258,14 @@ <h3>The <dfn><code>VisualViewport</code></dfn> interface</h3>
                 Although it is referred to as the pinch-zoom scaling factor, it can be affected through means other than pinch-zooming. e.g. When the UA centers and zooms in a focused edit box.
               </div>
             </dd>
+            <dt><dfn>segments</dfn></dt>
+            <dd>
+              Returns an array of DOMRects that represent the dimensions of each existing viewport segment. Returns null when called from within an iframe context. 
+            </dd>
+            <div class="note">
+              Based on the data returned for each viewport segment, developers will be able to infer the number of hinges available as well as the hinge orientation.
+               A user may at any point take the browser window out of spanning mode and place it on one of the screens or vice-versa, in those cases the window resize event will fire, and authors can query and get the number of available screen segments. 
```

Comment:
Maybe change the wording here to be more generic about the moving/resizing:
"The browser window can be moved/resized such that the number of segments and/or their dimensions change. In these cases, resize events will fire at which point authors should re-query this property, as the returned FrozenArray is just a snapshot of the current state"